knative/deployer.go

```diff
@@ -349,8 +349,8 @@ func updateService(f fn.Function, newEnv []corev1.EnvVar, newEnvFrom []corev1.En
 //     value: {{ env:MY_ENV }}
 func processLabels(f fn.Function) (map[string]string, error) {
 	labels := map[string]string{
-		"boson.dev/function": "true",
-		"boson.dev/runtime":  f.Runtime,
+		"function.knative.dev":         "true",
+		"function.knative.dev/runtime": f.Runtime,
```

Comment:
Should these use `labelFunctionKey` and friends?

Reply:
Yeah, we should have one place for labels. But I didn't want to do this big refactoring in this PR. Happy to do that in a follow up one, once we agree on this change.